src/main/resources/META-INF/plugin.xml

```diff
@@ -11,6 +11,7 @@
 
     <incompatible-with>com.intellij.jetbrains.client</incompatible-with>
 
+    <depends>com.intellij.modules.json</depends>
```

Comment:
As suggested:
https://plugins.jetbrains.com/docs/intellij/api-changes-list-2024.html#json-plugin-new-20243

it fixes the issue. 

The other place suggests that the verifier may give us false positives:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/21479962703890-JSON-plugin-dependency-cannot-be-resolved

I tested the feature and it worked for me - I think we can ignore the verifier's complain (as suggested in the post answer).